### PR TITLE
coprocessor: Support push down like expression.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: coprocessor.go [hunk 1]
```diff
@@ -49,7 +49,8 @@ func supportExpr(exprType tipb.ExprType) bool {
 		tipb.ExprType_ColumnRef,
 		tipb.ExprType_And, tipb.ExprType_Or,
 		tipb.ExprType_LT, tipb.ExprType_LE, tipb.ExprType_EQ, tipb.ExprType_NE,
-		tipb.ExprType_GE, tipb.ExprType_GT, tipb.ExprType_NullEQ:
+		tipb.ExprType_GE, tipb.ExprType_GT, tipb.ExprType_NullEQ,
+		tipb.ExprType_Like:
```
Comment (shenli) [inline coprocessor.go]: Miss tipb.ExprType_Not. 
Not Like "xxx" will be convert to an UnaryExpr and a LikeExpr.

See: https://github.com/pingcap/tidb/pull/1080/files

**Comment by ngaut** [comment]
PTAL @shenli @coocood @BusyJay

**Comment by coocood** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**